### DISCUSSION TITLE
having the release version on power-on screen

### DISCUSSION THREAD

**Discussion by OE1MWW** [egzumer/uv-k5-firmware-custom#147] [Ideas]
fiddeling around witha UV-K5(8) and a UV-K5R
iit would be nice to have the the release version (for example: '0.18') on power-on screen.

Right now it's checksum (?)

**Comment by ErikS-web** [comment]
No it is not. It is a reference number to a build.
And is also used as a reference for test builds as a reference of the Official Releases.

You can find the relationship here [Releases](https://github.com/egzumer/uv-k5-firmware-custom/releases)

![Knipsel](https://github.com/egzumer/uv-k5-firmware-custom/assets/148579604/d54804fc-14fd-4196-a726-f8dc0b08f338)

**Reply by OE1MWW** [reply]
Erik, sorry if I did not explain. I am using two devices, uploading firmware as soon
as a new one comes out - but - interrupted by my wife ;-) - I wondered - what version
is now on that one ?  'Traditional' on any HT, there is a menu entry, or similar, to read
on the device, without browsing the release pages - what version is installed.

**Reply by ErikS-web** [reply]
If you have a 3rd receiver at hand... Then you can try whether the 1750 Hz is really a little louder. That should be 0.18 version  ;-)

**Comment by f5gkw** [comment]
You can see the release in the menu.

**Reply by ErikS-web** [reply]
Of the radio ...  Than show me ! ... and ? 

Than I will update the Wiki.

**Comment by egzumer** [comment]
I will add that at some point. Possibly soon.
The id that is showing up now is an id of the commit at which the FW was released.
E.g. a0d052b68b5809336af2c00c25188c68137c23f2

**Reply by OE1MWW** [reply]
0.19 => perfect! Thank You! Well done!

**Reply by ErikS-web** [reply]
Picture / photo ?

**Reply by jclagreca** [reply]
I appreciate this update as well.  Makes it easy to see what version i'm on!  Thank you.

**Comment by ErikS-web** [comment]
The picture...
![EGZUMER v0 19](https://github.com/egzumer/uv-k5-firmware-custom/assets/148579604/f964d556-8610-40a4-acbc-a60723429d01)
